### PR TITLE
gotify-cli: Add version 2.2.1 - gotify-server: Add version 2.1.4

### LINKED ISSUE #1011: [Request] add gotify server, gotify cli
Hi there. I would like to suggest two new cool apps to add in the project:

1. gotify server (https://github.com/gotify/server/tree/v2.0.14)
2. gotify cli (https://github.com/gotify/cli)

### PR DESCRIPTION
**Pull request by graphixillusion**
Adding this two new app manifest:

gotify-cli: version 2.2.1
gotify-server: version 2.1.4

Closes #1011

### CODE REVIEW

File: bucket/gotify-cli.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/gotify/cli/releases/download/v$version/gotify-cli-windows-amd64.exe#/gotify-cli.exe"
+            },
+			"32bit": {
+				"url": "https://github.com/gotify/cli/releases/download/v$version/gotify-cli-windows-386.exe#/gotify-cli.exe"
+			}
+		}
+    }
+}
```
Comment (rashil2000) [inline bucket/gotify-cli.json:30]:
```suggestion
}

```

File: bucket/gotify-server.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/gotify/server/releases/download/v$version/gotify-windows-amd64.exe.zip"
+            },
+			"32bit": {
+				"url": "https://github.com/gotify/server/releases/download/v$version/gotify-windows-386.exe.zip"
+			}
+		}
+    }
+}
```
Comment (rashil2000) [inline bucket/gotify-server.json:41]:
```suggestion
}

```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/gotify-cli.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "2.2.1",
+    "description": "A command line interface for pushing messages to gotify/server",
```
Comment (rashil2000) [inline bucket/gotify-cli.json:3]:
```suggestion
    "description": "A command line interface for pushing messages to Gotify server",

```

File: bucket/gotify-server.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.1.4",
+    "description": "A simple server for sending and receiving messages in real-time per WebSocket. (Includes a sleek web-ui)",
```
Comment (rashil2000) [inline bucket/gotify-server.json:3]:
```suggestion
    "description": "A simple server for sending and receiving messages in real-time per WebSocket, with a sleek web UI",

```

File: bucket/gotify-server.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            "url": "https://github.com/gotify/server/releases/download/v2.1.4/gotify-windows-386.exe.zip",
+            "hash": "d3073247f1d6164362dfb1d279e40d664e3209672e6993763853defb561bb88a",
+            "bin": [
+                [
+                    "gotify-windows-386.exe",
+                    "gotify-server"
+                ]
+            ]
+        }
+    },
```
Comment (rashil2000) [inline bucket/gotify-server.json:27]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/gotify/server/releases/download/v2.1.4/gotify-windows-amd64.exe.zip",
            "hash": "adf8bdad8ec032325169b0dbaa5952bc13fb547b570532cef60f2007623fcc93"
        },
		"32bit": {
            "url": "https://github.com/gotify/server/releases/download/v2.1.4/gotify-windows-386.exe.zip",
            "hash": "d3073247f1d6164362dfb1d279e40d664e3209672e6993763853defb561bb88a"
        }
    },    
    "pre_install": "Rename-Item \"$dir\\gotify-windows-*\" gotify.exe",
    "bin": "gotify.exe",
```

We can simplify this a bit

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
This is your first contribution, right? Excellent manifests, btw! Just some minor comments.

**Comment by graphixillusion** [comment]
> This is your first contribution, right? Excellent manifests, btw! Just some minor comments.

Yep! First one. Thank you! I should do something else?

**Comment by rashil2000** [comment]
> I should do something else?

Check the review comments above (5 in total).

File: bucket/gotify-cli.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "2.2.1",
+    "description": "A command line interface for pushing messages to Gotify server",
+
```
Comment (rashil2000) [inline bucket/gotify-cli.json:4]:
```suggestion
```

File: bucket/gotify-server.json [hunk 7]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "2.1.4",
+    "description": "A simple server for sending and receiving messages in real-time per WebSocket, with a sleek web UI",
+
```
Comment (rashil2000) [inline bucket/gotify-server.json:4]:
```suggestion
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Remove empty newlines

**Pull request merged**